### DISCUSSION TITLE
Not Working After Build

### DISCUSSION THREAD

**Discussion by kecoliva** [TanStack/table#4602] [Q&A] [answered]
I'm using the latest version of Next.js with the latest version of TanStack/table.

Everything works fine when running the app in development via `next dev` but when trying to run the build version via `next build` and `next start` the table now produces this error:

![image](https://user-images.githubusercontent.com/46006869/207225858-959f591b-5dfc-4d02-bccc-6f45ec65f847.png)
![image](https://user-images.githubusercontent.com/46006869/207225917-299b6192-09ad-450c-994f-d95e62168b1c.png)

I'm only following the instructions on the docs on how to make a controlled pagination. Are there any compatibility issues with Next and TanStack/table? Or even the Node version?

**Comment by desiboli** [comment]
@kecoliva I had the same error! After setting "swcMinify" to false in in next.config.js file it all worked again! Not sure why swcMinify is breaking the table all of a sudden. 🤷‍♂️

**Reply by ElTupac** [reply]
Thanks for this, i will sacrifice this while not having a solution for the problem. I literally started having this problem from a moment to another, without touching code using the table at all.
I turned on the swcMinify flag of the next.config about two months now, it was impossible for me to link one thing with another.

**Comment by KevinVandy** [comment]
What versions of Next and Table are you using? Did this happen after package upgrades?

**Reply by kecoliva** [reply]
Things didn't worked after upgrading Next to 13.0.6 (from 13.0.5) and TanStack/table to 8.7.2 (from 8.7.0)

**Reply by ElTupac** [reply]
I am in Next 12.3.1 and Tanstack/react table in 8.5.15, and since yesterday my productions build started having the same issue as described, without touching code that is using the library.

**Reply by KevinVandy** [reply]
> I am in Next 12.3.1 and Tanstack/react table in 8.5.15, and since yesterday my productions build started having the same issue as described, without touching code that is using the library.

8.5.15 had known cjs problems. Upgrade to V8.7.0

**Comment by ShaunSpringer** [comment, marked as answer]
Having the same problem in `next` 13.0.x but only after we upgraded to `react-table` 8.7.3. It works fine on 8.7.0

**Comment by litewarp** [comment]
I had this problem on `8.7.0` and tried upgrading to `8.7.3` but it persists. On nextjs `13.0.x` as well. Will try disabling swcMinify

**Reply by kecoliva** [reply]
@litewarp completely remove `@tanstack/react-table` from your dependencies. Your version is probably `^8.7.0` (emphasis on the `^`). I didn't have to change anything to fix it, I simply installed `@tanstack/react-table@8.7.0`. I don't recommend setting `swcMinify` to `false`.

**Reply by litewarp** [reply]
@kecoliva That worked! Thanks
